code/modules/recycling/disposal/outlet.dm

```diff
@@ -1,12 +1,16 @@
-//how fast disposal machinery is ejecting things (does not effect range)
+//how fast disposal machinery is ejecting things and how far it goes
 /// The slowest setting for disposal eject speed
 #define EJECT_SPEED_SLOW 1
+#define EJECT_RANGE_SLOW 2
 /// The default setting for disposal eject speed
 #define EJECT_SPEED_MED 2
+#define EJECT_RANGE_MED 4
```

Comment:
In testing, 4 is too much for medium ejection range since it's the default range and mapped-in disposals systems are built around having 2 tiles. Might be good to just keep it at 2 and make slow ejection have only 1, since map changes are likely out-of-scope for this PR.